cuda_core/tests/helpers/buffers.py

```diff
@@ -0,0 +1,122 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import ctypes
+import sys
+
+from cuda.core.experimental import Buffer, MemoryResource
+from cuda.core.experimental._utils.cuda_utils import driver, handle_return
+
+if sys.platform.startswith("win"):
```

Comment:
We should migrate this to a common place so that this pattern doesn't get replicated in multiple files.  It would be a maintenance headache if we had to update it in multiple places.